tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/TestMethodAttributeExecuteIntegration.cs

```diff
@@ -145,8 +171,7 @@ internal static CallTargetState OnMethodBegin<TTarget, TTestMethod>(TTarget inst
                 if (remainingRetries > 0)
                 {
                     // Handle retries
-                    var results = new List<IList>();
-                    results.Add(returnValueList);
+                    var results = new List<IList> { returnValueList };
```

Comment:
Why not go all the way 😄 
```suggestion
                    List<IList> results = [ returnValueList ];
```